DFC.App.FindACourse/Framework/CorrelationIdProvider.cs

```diff
@@ -9,7 +9,7 @@ namespace DFC.App.FindACourse.Framework
 {
     public class CorrelationIdProvider : ICorrelationIdProvider
     {
-        private const string CorrelationId = "DssCorrelationId";
+        private const string CorrelationId = "CorrelationId";
```

Comment:
Is this constant required? can you get this string from the CorrelationId Nuget (header string?)